opal/util/info.c

```diff
@@ -85,6 +87,16 @@ int opal_info_dup(opal_info_t *info, opal_info_t **newinfo)
     return OPAL_SUCCESS;
 }
 
+int opal_info_dup_public(opal_info_t *info, opal_info_t **newinfo)
```

Comment:
Move these into a header file and make them always inline.

Reply:
I'm hesitant about this since there are no other implementations in `info.h`. The call to `opal_info_dup_impl` will be a `jmp` so there is little to gain from inlining these.

Reply:
A call into a function that does a simple jmp ? I don't even see the interest of having these functions, with a weird name, instead of just using `opal_info_dup` with all 3 arguments.